boot/assets.go

```diff
@@ -354,6 +359,23 @@ func (o *TrustedAssetsInstallObserver) ChosenEncryptionKeys(key, saveKey keys.En
 	o.saveEncryptionKey = saveKey
 }
 
+func (o *TrustedAssetsInstallObserver) UpdateBootEntry() error {
```

Comment:
changes in assets.go don't have matching unit tests afaict

Reply:
There should be `TestUpdateBootEntryOnInstall` and `TestUpdateBootEntryOnUpdate` in `boot/assets_test.go`.

Reply:
Ah sorry, that was an older comment.